internal/validate/vsa/vsa.go

```diff
@@ -315,7 +376,7 @@ func (c *VSAChecker) CheckExistingVSA(ctx context.Context, imageRef string, expi
 		"expiration_threshold": expirationThreshold,
 		"expired":              result.Expired,
 		"verifier":             predicate.Verifier,
-		"policy_source":        predicate.PolicySource,
+		"policy_source":        predicate.Policy.Name,
```

Comment:
I don't understand this change. Mistake?

Reply:
Yes, good catch. Left over code.